nss-cohort-36/keahua-arboretum-pixel-pals
# Lr modules for release animal

# Description

Made a module for the release animal function. User should be able to create animals and add them to biomes

Fixes #61 (issue)

## Type of change

Please delete options that are not relevant.



# Testing Instructions

1. `git fetch --all`
2. `git checkout lr-modules-for-release-animal`
3. `python index.py`
4. User should be able to create animals and add them to biomes. Make sure to type an option that is not on the list when adding the animal to a biome to test the except functionality. 



# Checklist:
